Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -116,6 +120,11 @@ void FullscreenManager::requestFullscreenForElement(Ref<Element>&& element, RefP
     // If any of the following conditions are true, terminate these steps and queue a task to fire
     // an event named fullscreenerror with its bubbles attribute set to true on the context object's
     // node document:
+    if (!element->isHTMLElement() && !is<SVGSVGElement>(element) && !is<MathMLMathElement>(element)) {
```

Comment:
This broke the PlayStation build, as EWS would have shown (more specifically, it's the `!ENABLE(MATHML)` that you broke).